sdk/src/psp22.ts

```diff
@@ -90,7 +93,19 @@ export class PSP22 {
     )
   }
 
-  async mint(account: IKeyringPair, value: bigint, block: boolean = true): Promise<TxResult> {
+  async load(address: string) {
```

Comment:
`load` naming creates collisions with an already existing loading. To make it clear I suggest change naming. Maybe: `changeContractAddress`